core-web/libs/sdk/angular/README.md

```diff
@@ -0,0 +1,7 @@
+# sdk-angular
```

Comment:
Why, What, and How use it. In this file. Only for no forgetting it in the future.